AB#2512 Config secrets via env var & config refactoring (#544)
edgelesssys/constellation: CHANGELOG.md

## [Unreleased]

### Added

> - `CONSTELL_AZURE_CLIENT_SECRET_VALUE` as an alternative to provide `provider.azure.clientSecretValue` via environment variable instead.

Reviewer: The sentence reads a bit odd. Maybe
```suggestion
Environment variable `CONSTELL_AZURE_CLIENT_SECRET_VALUE` as an alternative way to provide the configuration value `provider.azure.clientSecretValue`.
```
